src/Microsoft.Framework.PackageManager/Program.cs

```diff
@@ -256,6 +257,37 @@ public int Main(string[] args)
                 });
             });
 
+            app.Command("list", c =>
+            {
+                c.Description = "Print the dependencies of a given project.";
+                var coreclrRoot = c.Option("--coreclr",
```

Comment:
This is not a switch, it accept a path to the CoreCLR assemblies folder. The usage is different from the `-r` and `--runtime`.